### PR TITLE
Add AWS package

### PR DESCRIPTION
**Pull request by ruflin**
This package is taken from the aws module. It is used for a more complex example.

### CODE REVIEW

**Comment by ruflin** [comment]
@jen-huang Currently when I try to add this as data source I get:

```
TypeError: Cannot read property 'length' of undefined
    at DatasourceInputPanel (http://localhost:5601/bundles/plugin/ingestManager/1.plugin.js:12494:35)
    at renderWithHooks (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:466737:18)
    at mountIndeterminateComponent (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:469271:13)
    at beginWork$1 (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:470639:16)
    at HTMLUnknownElement.callCallback (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:450813:14)
    at Object.invokeGuardedCallbackDev (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:450862:16)
    at invokeGuardedCallback (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:450917:31)
    at beginWork$$1 (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:476257:7)
    at performUnitOfWork (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:475175:12)
    at workLoopSync (http://localhost:5601/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:475148:22)
```

I assume I have a typo or something similar somewhere.

File: dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        default: 5m
+
+# TODO: The configs for the cloudwatch are pretty complex, arrays of configs.
+
+
+#  metrics:
+#    - namespace: AWS/EC2
+#      #name: ["CPUUtilization", "DiskWriteOps"]
+#      tags.resource_type_filter: ec2:instance
+#      #dimensions:
```
Comment (ruflin) [inline dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml]: @jen-huang This is the part where I think it becomes tricky to build a UI around it. We have an array with values inside. I think here we should allow the user to add his own yaml (in the future) for this property. Something like offer a textarea?

**Comment by ruflin** [comment]
@jen-huang Above error is now gone, not sure how I fixed it :-)

**Comment by ruflin** [comment]
Merging this PR so we have one more package available for testing. It does not seem to break anything else in EPM.

File: dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml (hunk 1 above)
Comment (jen-huang) [reply dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml]: Let's chat about this as I'm not 100% clear on this

Comment (ruflin) [reply dev/packages/example/aws-1.0.0/dataset/cloudwatch/manifest.yml]: Lets try to sync up later today on it.

**Pull request merged**